stdlib/importlib/metadata/_meta.pyi

```diff
@@ -17,7 +16,6 @@ class SimplePath(Protocol):
     def joinpath(self) -> SimplePath: ...
     def parent(self) -> SimplePath: ...
     def read_text(self) -> str: ...
-    if sys.version_info >= (3, 11):
-        def __truediv__(self) -> SimplePath: ...
-    else:
-        def __div__(self) -> SimplePath: ...
+    # There was a bug in `SimplePath` definition in cpython, see #8451
+    # Previously `__div__` was used, but it should have been `__truediv__`.
```

Comment:
```suggestion
    #  Strictly speaking `__div__` was defined in 3.10, not __truediv__,
    # but it should have always been `__truediv__`.
```

Reply:
Shouldn't `in 3.10` be `before 3.10`?

Reply:
No — the class is new in 3.10, and has the `__div__` method in 3.10. In 3.11, the method has been changed to `__truediv__`, but it hasn't been backported to 3.10.